WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java

```diff
@@ -418,6 +422,10 @@ public int getPagingCount() {
     public @NonNull String getPagingDescription() {
         if (mActivity == null) return "";
 
+        if (!getShouldPageComments()) {
+            return mActivity.getString(R.string.none);
```

Comment:
I think 'disabled' makes more sense here and chatted with @mattmiklic about it. He agrees, let's switch it to 'Disabled'.